mteb/cli/build_cli.py

```diff
@@ -361,6 +361,72 @@ def _add_create_meta_parser(subparsers) -> None:
     parser.set_defaults(func=_create_meta)
 
 
+def _add_leaderboard_parser(subparsers) -> None:
+    parser = subparsers.add_parser("leaderboard", help="Launch the MTEB leaderboard")
+
+    parser.add_argument(
+        "--cache-path",
+        type=str,
+        help="Path to the cache folder containing model results",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "--host",
+        type=str,
+        default="127.0.0.1",
```

Comment:
shouldn't default be None?

Reply:
But `leaderboard/app.pyL#1043` has the following: 
```python
server_name="0.0.0.0",
```